File: desafio1.js
// Factorizacion

let originalNum = 60;
let num = 60;
let primos = [2,3,5,7,11,13];
let sum = 0;

for(let i=0; i<primos.length; i++){
    if(num%primos[i] == 0){
        sum += num/primos[i];
        num = num/primos[i];
        i = -1;
    }
}

console.log(sum+originalNum);

File: desafio1/desafio1.cpp
#include <iostream>
#include <string>
#include <algorithm>

using namespace std;

int main() {
    const int originalNum = 60;
    int num = 60;
    int primos[11] = {2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31};
    int sum = 0;

    for (int i = 0; i < 11; i++) {
        while (num % primos[i] == 0) {
            sum += num / primos[i];
            num = num / primos[i];
        }
    }

    cout << sum + originalNum;
    return 0;
}